u6-spas/src/router/index.ts

```diff
@@ -0,0 +1,39 @@
+import Vue from 'vue'
+import VueRouter, { RouteConfig } from 'vue-router'
+import Home from '@/views/Home.vue'
+import Colors from '@/views/Colors.vue'
+import Profile from '@/views/Profile.vue'
+import Converter from '@/views/Converter.vue'
+
+Vue.use(VueRouter)
+
+const routes: Array<RouteConfig> = [
+  {
+    path: '/',
+    name: 'Home',
+    component: Home
+  },
+  {
+    path: '/colors',
+    name: 'Colors',
+    component: Colors
+  },
+  {
+    path: '/profile/edit',
+    name: 'Profile',
```

Comment:
ในโครงการจริง มีแนวโน้มว่าคุณจะมีหน้าอื่นเพื่อดูโปรไฟล์ ฉันมักจะตั้งชื่อเส้นทางของฉันตามอ็อบเจกต์ + การกระทำ โดยให้ชื่อออบเจกต์ก่อน เพื่อให้รวมกลุ่มเข้าด้วยกัน เช่น
On a real-project, it's likely you'll have another page to view the profile. I usually name my routes based on the object + action, with the object first to group them together, for example:

- ProfileView
- ProfileEdit
- ProfileList
- ...etc.